src/controller/motor.c

```diff
@@ -657,12 +643,12 @@ void TIM1_CAP_COM_IRQHandler(void) __interrupt(TIM1_CAP_COM_IRQHANDLER)
   }
   else // nothing to limit, so adjust duty_cycle to duty_cycle_target, including ramping
   {
-    // if brake is active, keep duty cycle target at 0
-    if (ui8_motor_controller_state & MOTOR_CONTROLLER_STATE_BRAKE) { ui8_controller_duty_cycle_target = 0; }
```

Comment:
Are you sure removing this check is safe ?

Reply:
WIP.